### PR TITLE
Add new styling to tables and blockquote

### PR DESCRIPTION
**Pull request by CalebJohn**
This updates tables and blockquotes to have a bit more style. I guess that a designer could come up with something better, but I feel these are an improvement on the current styles.

image

image

### CODE REVIEW

**Comment by tessus** [comment]
Does this mean that all tables are now striped? Personally I truly dislike striped tables and in most cases I want them to look like this:

image

Also where are the vertical lines in your tables?

IMO, there should be a UI for choosing how a table or element should look like. Changes like this are affecting a huge audience and changing elements on a global level is basicalling forcing the change onto everyone.

**Comment by laurent22** [comment]
It's true that any styling change is going to affect many users, but we need to find a reasonably good default that both looks nice and is usable.

I see many Markdown viewer have tables with stripes because it makes it easier to read tables with long rows so I think we should have this as well. On the other hand, compared to Typora for instance (see below), the gray colours is a bit too dark, and also we don't need a darker colour for the heading because it's already bold text.

So basically I think we can dim the effect a bit. I agree we also need to keep the cell borders, and text should be left justified by default.

image

For customisation, I think we already have userstyle.css for this and that would work for all tables, right?

**Comment by subi54** [comment]
@laurent22 userstyle.css does work with tables. It'll have to take precedence over the default one. For people who need it that is.

**Comment by CalebJohn** [comment]
Just below is the updated table based on your recommendations. I added a 2px border underneath the heading because I don't find that bolding is enough to separate headings from content. Below I've included a github table just to compare the looks. I understand that changing default styles can be a bit controversial, so I'm happy to keep changing this until everyone is happy.
@tessus if you still dislike the striped tables they can be removes using userstyle.css with 3 lines
```css
td {
    background-color: white;
}
```
#### Joplin
image


#### Github
| Name | Value|
| --- | --- |
| test | 1 |
| some | thing |
| more | stuff |

**Comment by tessus** [comment]
@CalebJohn thanks for the info.

**Comment by laurent22** [comment]
@CalebJohn, for the heading I meant we can make the dark gray lighter (the same colour as the other gray rows) but I didn't mean to remove the gray background completely. I think it looks better with it. A bit like the example below:

image

If you could just tweak this that would be great. Other than that I like the changes and we can add it to the next release since it's related to the other CSS changes.

**Comment by CalebJohn** [comment]
@laurent22 Okay I added that, should be good to go!

**Comment by laurent22** [comment]
Looks good, thanks @CalebJohn!
